download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -241,11 +272,14 @@ def join_hts(datasets, coverage_datasets=[], reference_genome='37'):
     joined_ht.write(os.path.join(output_path))
 
 def run():
-    # TODO: '38' when gnomad liftover is done.
```

Comment:
Might make sense to have the user specify which genome version. Usually we use args, but we can consider reading in input, depends on what you think is better ux https://www.geeksforgeeks.org/taking-input-in-python/.

Reply:
I'm good with using argparse if you are? I like the idea of kicking it off and not worrying about it. Plus, I think user input into dataproc is tricky.